src/routes/+page.svelte

```diff
@@ -193,26 +194,63 @@
 			<h5>{$_('plannedDeposits')}</h5>
 			<a href={routes.DEPOSIT()}>+</a>
 		</div>
-		{#each detailStore.deposits as deposit, i}
-			<div>
-				{JSON.stringify(deposit)}<a href={routes.DEPOSIT(i)}>{$_('edit')}</a><button
-					onclick={() => detailStore.removeDeposit(i)}>{$_('delete')}</button
-				>
-			</div>
-		{/each}
+		<div class="grid">
+			{#each detailStore.deposits as deposit, i}
+				<div class="flex-container">
+					<div class="info-of-operation">
+						<p class="operation-name">{deposit.name}</p>
+						<p class="operation-amount">
+							{deposit.amount}&nbsp;{detailStore.currency}{deposit.isRecurring
+								? ' / ' + deposit.frequency
```

Comment:
Please use translation for the frequency. Right now it is `day`, `month`, `year` in both czech and english.